main.py

```
@@ -1,8 +1,11 @@
+##########################################################################
+###################### First Latvian Fasker' Ripper ######################
+#################### Copyright (c) 2024-2025 mr.Iceman ###################
+##########################################################################
 from logging.handlers import TimedRotatingFileHandler
 
 from config import *
```

Ruff (0.8.2)

`from config import *` used; unable to detect undefined names

(F403)